cmd/tsidp/Dockerfile

```diff
@@ -1,41 +1,63 @@
-# Build stage
-FROM golang:alpine AS builder
+# Copyright (c) Tailscale Inc & AUTHORS
```

Comment:
I would argue that we need to remove this Dockerfile altogether.

Two mechanisms introduce a risk of drift when only one mechanism is updated and this PR illustrates that - the Docker run commands added here don't work with an image that's built with `mkctr`.